src/app/pages/home-page/home-page.component.scss

```diff
@@ -22,3 +21,9 @@
 .drop-box {
   width: 100%;
 }
+
+.drop-box {
+  ::ng-deep .mat-form-field-wrapper {
+    padding-bottom: 0;
+  }
+}
```

Comment:
This style here becomes global after using ng:deep.
Are you sure it is not affecting other places?